### ISSUE TITLE
Updating Geexo WL token and adding Geexo OG token

### ISSUE THREAD

**Issue by GeexoYokai** [solana-labs/token-list#24716]
- **I'm submitting a ...**
  [X] update request

### Summary

**TLDR;**
_Requesting changes in this pull-request -> https://github.com/solana-labs/token-list/pull/24714_

I am updating the **Geexo WL** that I previously added in this [pull-request](https://github.com/solana-labs/token-list/pull/23327) ([commit](https://github.com/solana-labs/token-list/pull/23327/commits/cf232af9925eaa37e284cbdded42c7efcf22a73c)).

[Pull-request updating Geexo WL](https://github.com/solana-labs/token-list/pull/24714)
[Pull-request where Geexo WL was originally created](https://github.com/solana-labs/token-list/pull/23327)

Expected changes:

- Rename current **Geexo WL** to **Geexo OG**
- Replace current **Geexo WL** logo with **Geexo OG**'s
- Create a new token named **Geexo WL**
- Move **Geexo WL** logo to new token

**Comment by GeexoYokai** [comment]
Hey @keone hope everything is well -- would you merge this whenever you have time?

Thanks in advance!
